### PR TITLE
Stress Tests 2 - QA Tests

### LINKED ISSUE #5563: Stress Tests 2 - QA Tests
Stress Tests ( These tests should be automated but not integrated in github CI actions, so we can invoke these tests locally any time when required ) 

- [x] Sending multiple requests in parallel with large payload
- [x] Response with large result data size on WS and IPC ( data sent back in multiple chunks )

### PR DESCRIPTION
**Pull request by avkos**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

Fixes #5563

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: scripts/stressTests/data/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const { Web3 } = require('../../../packages/web3');
```
Comment (jdevcs) [inline scripts/stressTests/data/index.js]: move stress tests under web3 tests dir instead of in scripts

**Review by jdevcs — COMMENTED** [review]
lgtm, just need above change

**Review by luu-alex — APPROVED** [review]

**Review by Muhammad-Altabba — APPROVED** [review]
I added the review points t https://github.com/web3/web3.js/pull/6595 as it already contains the commits of this PR.

File: packages/web3/test/stress/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		arguments: [123, ''],
+		gasPrice: await web3.eth.getGasPrice(),
+		gas: BigInt(9000000000000),
+		gasLimit: BigInt(9000000000000),
+		type: BigInt(0),
+	};
+	const c = new web3.eth.Contract(contractData.abi);
+	const contract = await c.deploy(deployOptions).send(sendOptions);
+
+	console.time(`Send huge data [${i}]`);
```
Comment (jdevcs) [inline packages/web3/test/stress/index.js]: same here as shared in https://github.com/web3/web3.js/pull/6595#discussion_r1398925070

**Review by jdevcs — APPROVED** [review]
approved, for improvements you can do in other wip Stress testing PR or followup PR/issue.

**Pull request merged**